e2e/tests/features/202-cuesheet.spec.ts

```diff
@@ -1,11 +1,20 @@
-import { test } from '@playwright/test';
+import { expect, test } from '@playwright/test';
 
-test('cuesheet displays events and exports csv', async ({ page }) => {
+test('cuesheet displays events', async ({ page }) => {
   // same elements in cuesheet
   await page.goto('http://localhost:4001/cuesheet');
-  await page.getByText('Eurovision Song Contest').click();
-  await page.getByRole('cell', { name: 'Lunch break' }).click();
-  await page.getByRole('cell', { name: 'Albania' }).click();
-  await page.getByRole('cell', { name: 'Latvia' }).click();
-  await page.getByRole('cell', { name: 'Lithuania' }).click();
+  await expect(page.getByText('Eurovision Song Contest')).toBeVisible();
+  await expect(page.getByRole('row', { name: 'Lunch break' })).toBeVisible();
+
+  await expect(
+    page.getByRole('row', { name: 'SF1.01 10:00:00 10:20:00 00:20:00' }).getByRole('textbox').first(),
+  ).toHaveValue('Albania');
```

Comment:
any changes to the data would make this tests break. we need something a bit more generic that wont be so painful to maintain